File: exercise2-pyramid-of-odd-numbers.py
x = int(input("Enter a number: "))

while x <= 0:
    print("Please enter a valid positive integer!")
    x = int(input("Enter a number: "))

for i in range(x):
    num = 1
    for j in range(i + 1):
        print(num, end=" ")
        num = num + 2
    print()

File: exercise2-pyramid-of-odd-numbers.js
var x = parseInt(prompt("Enter a number"));

for (var i = 0; i <= x; i++){
    var num = 1;
    var line = ""
    for(var j = 0; j <= i; j++){
        line += num.toString() + " ";
        num += 2;
    }
    console.log(line);
}